cmd/lakectl/cmd/local.go

```diff
@@ -146,6 +147,22 @@ func localHandleSyncInterrupt(ctx context.Context) context.Context {
 	return ctx
 }
 
+func dieOnInterruptedOperation(interruptedOperation string, force bool) {
+	if !force {
+		switch strings.ToLower(interruptedOperation) {
+		case "commit":
+			Die(`Latest commit operation was interrupted, remote data may be incomplete.
+	Use "lakectl local commit... --force" to commit your latest changes.`, 1)
```

Comment:
Requiring a commit operation to fix an operation interrupt issue is not a valid request from the user IMO.
In this case we might want to just let them know and decided for themselves whether they want to commit again (no need for force) or maybe to pull changes from the server in case of a suspected corrupt download

Reply:
@N-o-Z So should we show a message that indicates that the latest operation went wrong? (same question for the checkout operation).